geowebcache/rest/src/main/java/org/geowebcache/rest/controller/ReloadController.java

```diff
@@ -54,14 +57,30 @@
 
     @RequestMapping(value = "/reload", method = RequestMethod.POST)
```

Comment:
We should maybe have an `accepts=application/x-www-form-urlencoded` here, but since this is the only implementation for this endpoint I think it is fine as-is.